pkg/db/postgres.go

```diff
@@ -1,8 +1,12 @@
 package db
```

Comment:
the db should be isolated/encapsulated for example the DB package should have not knowledge of http requests or redendering the data to Browser, that should handled by caller